models/perpetual/perpetual_trades.sql

```diff
@@ -1,5 +1,10 @@
 {{ config(
         alias ='trades',
+        partition_by = ['block_date'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['block_time', 'tx_hash', 'evt_index'],
```

Comment:
we can ignore for this PR, since these are still unique, but noting to self to potentially revisit unique keys on this sector-wide spell and it's upstream spells to match closer to `dex.trades` (likely without `trace_address`)
https://github.com/duneanalytics/spellbook/blob/main/models/uniswap/ethereum/uniswap_v1_ethereum_trades.sql#L8